WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1045,8 +1045,8 @@ public void initializeEditorFragment() {
         if (mEditorFragment instanceof AztecEditorFragment) {
             AztecEditorFragment aztecEditorFragment = (AztecEditorFragment)mEditorFragment;
             aztecEditorFragment.setEditorBetaClickListener(EditPostActivity.this);
-            aztecEditorFragment.setAztecImageLoader(new AztecImageLoader(getBaseContext()));
-            aztecEditorFragment.setAztecVideoLoader(new AztecVideoLoader(getBaseContext()));
+            aztecEditorFragment.setAztecImageLoader(new AztecImageLoader(getBaseContext(), aztecEditorFragment.loadingImagePlaceholder));
```

Comment:
`loadingImagePlaceholder` and `loadingVideoPlaceholder` won't exist until `onCreateView` of `AztecEditorFragment` is called, so I have a feeling this might lead to some unforeseen consequences in a future. Maybe we could do lazy loading here? What do you think?

Reply:
Good point, I will make the host activity to create and pass placeholders to Aztec Editor(s).

Reply:
I ended up with a solution where the host activity creates the loader placeholders and pass them to the fragment and to the loaders.